packages/nextjs/components/example-ui/ContractInteraction.tsx

```diff
@@ -63,15 +63,16 @@ export const ContractInteraction = () => {
             <div className="flex rounded-full border border-primary p-1 flex-shrink-0">
               <div className="flex rounded-full border-2 border-primary p-1">
                 <button
-                  className={`btn btn-primary rounded-full capitalize font-normal font-white w-24 flex items-center gap-1 hover:gap-2 transition-all tracking-widest ${
-                    isLoading ? "loading" : ""
-                  }`}
+                  className="btn btn-primary rounded-full capitalize font-normal font-white w-24 flex items-center gap-1 hover:gap-2 transition-all tracking-widest"
                   onClick={() => writeAsync()}
+                  disabled={isLoading}
                 >
-                  {!isLoading && (
+                  {!isLoading ? (
                     <>
                       Send <ArrowSmallRightIcon className="w-3 h-3 mt-0.5" />
                     </>
+                  ) : (
+                    <span className="loading loading-spinner loading-sm"></span>
```

Comment:
Oh, I see, now you are showing this. OK.

Reply:
Yeah actually they have removed using `loading` class directly and suggest to this now checkout [here](https://daisyui.com/components/button/#button-with-loading-spinner) 🙌